site/docs/how-to-release.md

```diff
@@ -311,74 +311,9 @@ Create a PR in the `iceberg` repo to make revapi run on the new release. For an
 ### Documentation Release
 
 Documentation needs to be updated as a part of an Iceberg release after a release candidate is passed.
-The commands described below assume you are in a directory containing a local clone of the `iceberg-docs`
-repository and `iceberg` repository. Adjust the commands accordingly if it is not the case. Note that all
-changes in `iceberg` need to happen against the `master` branch and changes in `iceberg-docs` need to happen
-against the `main` branch. 
+Please follow the steps from [README](../README.md) to update the site docs and javadocs.
```

Comment:
should this link to the section that mentions what needs to be done during a release?

Reply:
I wanted to link to https://github.com/apache/iceberg/blob/main/site/README.md#release-process, but `release deploy` won't work. So, I linked the whole file. 

P.S: I just want to get rid of referencing the archived `iceberg-docs` repo. Once we have automation we can update this section or add more info about manual steps in a follow up.

Reply:
Ajantha, now that the release is out, could you and @Fokko and @nastra help me review my release script?

Three thinks I'd like to have fixed by the next release are:

1. Fix internal links that cross versioned doc and site boundaries.

1. Add release script.

1. Update release and contribution docs.

Reply:
Always happy to validate. 

So you want us to keep this PR on hold until we have the script for release?